pkgs/development/tools/protoc-gen-grpc-web/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, protobuf }:
+
+stdenv.mkDerivation rec {
+  pname = "protoc-gen-grpc-web";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "grpc";
+    repo = "grpc-web";
+    rev = version;
+    sha256 = "sha256-NBENyc01O8NPo84z1CeZ7YvFvVGY2GSlcdxacRrQALw=";
+  };
+
+  # remove once PR merged
+  # https://github.com/grpc/grpc-web/pull/1107
+  patches = [
+    (fetchpatch {
+      name = "add-prefix";
+      url = "https://patch-diff.githubusercontent.com/raw/grpc/grpc-web/pull/1107.patch";
```

Comment:
Please use the URL for the commit, rather than for the PR.  (Because the PR is still open, it can be updated, and then the hash would break.)